arm-hdinsight/2015-03-01-preview/swagger/cluster.json

```diff
@@ -96,62 +96,12 @@
                 },
                 "x-ms-long-running-operation": true
             },
-            "patch": {
-                "tags": [
-                    "Clusters"
-                ],
-                "operationId": "Clusters_Update",
-                "description": "Patch HDInsight cluster with the specified parameters.",
-                "x-ms-examples": {
-                    "Patch HDInsight Linux clusters": {
-                        "$ref": "../examples/PatchLinuxHadoopCluster.json"

```

Comment:
If this operation is really no longer required then you should delete the corresponding example (I assume it's no longer required).